### PR TITLE
feat: Add intlify to the list of community partners

### LINKED ISSUE #173: [OTHER] Add a new community partners
### What would you like to share?

URL: https://github.com/intlify
Logo: https://github.com/intlify/art  
Description: Intlify is a project that aims to improve Developer Experience in software internationalization. We will aim to improve the Developer Experience of internationalization by providing libraries, frameworks, and tools that break down barriers to internationalization.

Can you choose a logo from the repository? ^

### Additional information

_No response_

### Would you like to work on this issue?

- [ ] Yes, I want to work on this issue!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by vinzvinci**
#### Description of the change

Closes #173 

#### Screenshots
image

#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/174"></a>

### CODE REVIEW

File: src/pages/community-partners.astro [hunk 1]
```diff
@@ -37,6 +37,13 @@ const partners = [
     'description': 'We believe that entrepreneurship should be open to all and that anyone with an idea deserves a way to innovate and grow.  Founders Hub helps remove traditional barriers to building a tech company by democratizing historically inaccessible resources including expert mentorship.',
     'extraClass': 'bg-white p-4 rounded-sm'
   },
+  {
+    'name': 'intlify',
+    'image': 'https://github.com/intlify.png',
+    'link': 'https://github.com/intlify',
```
Comment (Panquesito7) [inline src/pages/community-partners.astro:43]: Maybe link to their website instead?
```suggestion
    'link': 'https://intlify.dev/',
```

Comment (vinzvinci) [reply src/pages/community-partners.astro:43]: Yeah thanks, good idea!

**Review by Panquesito7 — APPROVED** [review]
Awesome! 🚀 Thank you! 😄👍

**Review by shuklaritvik06 — APPROVED** [review]
LGTM

**Review by krishguptadev — APPROVED** [review]

**Pull request merged**